katharsis-ui/package.json

```diff
@@ -1,18 +1,56 @@
 {
   "name": "katharsis-ui",
-  "version": "0.1.0",
-  "private": true,
-  "devDependencies": {
-    "react-scripts": "0.6.1"
+  "version": "0.0.0",
```

Comment:
This should be the same as the rest of the project. You probably will have to mark this file as templated and have maven's velocity engine do the work